templates/docs/whats-new.md

```diff
@@ -20,12 +20,12 @@ When we add, make significant updates, or deprecate a component we update their
     </tr>
   </thead>
   <tbody>
-    <!-- 2.35 -->
+    <!-- 2.36 -->
     <tr>
-      <th><a href="/docs/base/forms#validation">Forms / Validation</a></th>
-      <td><div class="p-label--updated">Updated</div></td>
-      <td>2.35.0</td>
-      <td>We've updated the styling of the form validation and required patterns</td>
+      <th><a href="/docs/patterns/accordion">Accordion / Tick elements</a></th>
```

Comment:
If we use a heading in the docs we should be able to link directly to it, but please check if the `id` is correct, I was suggesting it out of memory ;)

```suggestion
      <th><a href="/docs/patterns/accordion#with-tick-elements">Accordion / Tick elements</a></th>
```

Reply:
Added